utils/slice.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// SliceIntToString this will return slice into string example "[1,2,3,4,5]"
+func SliceIntToString(a []int64, delim string) string {
+	return strings.Trim(strings.Replace(fmt.Sprint(a), " ", delim, -1), "")
```

Comment:
Why not marshal the slice into json?

Reply:
@FilipNikolovski yes you are right i will change the field in db from varchar -> json